sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -679,3 +679,8 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 #IfMissingColumn form_vitals oxygen_flow_rate
 ALTER TABLE `form_vitals` ADD `oxygen_flow_rate` FLOAT(5,2) NULL DEFAULT '0.00';
 #EndIf
+
+#IfMissingColumn form_clinical_notes note_related_to
+ALTER TABLE `form_clinical_notes` ADD `note_related_to` TEXT NULL COMMENT 'Reference to lists id for note relationships(json)';
+ALTER TABLE `form_care_plan` ADD `note_related_to` TEXT NULL COMMENT 'Reference to lists id for note relationships(json)';
```

Comment:
drop the NULL in above 2 lines
(for text fields, we simplify things by not messing with default things)

Reply:
I see that you replenished your semicolons though :)

![2DV](https://user-images.githubusercontent.com/278968/121822139-4dd32580-cc52-11eb-8c37-38441c72f78e.gif)

Reply:
Yeah, they're getting hard to get. Plenty colons, periods, commas and smiley faces but, semicolons, big demand!